tests/test_relational.py

```diff
@@ -680,6 +680,12 @@ def test_legend_has_no_offset(self, long_df):
         for text in g.legend.texts:
             assert float(text.get_text()) > 1e7
 
+    def test_lineplot_2d_dashes(self, long_df):
+
+        ax = lineplot(data=long_df[["x", "y"]], dashes=[(5, 5), (10, 10)])
+        assert ax.get_lines()[0]._us_dashSeq == (5, 5)
+        assert ax.get_lines()[3]._us_dashSeq == (10, 10)
```

Comment:
Please do not access private attributes of third-party objects

Reply:
Solved thansk for the feedback!